### DISCUSSION TITLE
How can I find the owner of an org?

### DISCUSSION THREAD

**Discussion by rvirding** [community/community#102428] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I would like to find the owner of the org `luerl`. I have created a language with the name and would like to keep it in an org with the same name. Unfortunately there is absolutely no informatation at all for that org, it has no repos and has not been used over the last 6  years.

Any help would be much appreciated.

**Comment by naveen-98** [comment]
If you're looking to inquire about an organization on GitHub that seems inactive and you wish to use its name, and if the organization has no visible activity or contact information, you can follow these steps:

1. **GitHub Support:**
   - Contact GitHub Support using the [GitHub Support Contact Form](https://support.github.com/contact).
   - Choose a relevant topic, such as "Repositories" or "Organizations."
   - Clearly explain your situation, mentioning that you would like to inquire about the ownership or status of an inactive organization ("luerl") that you are interested in using for a project.

2. **Include Details:**
   - Provide details about your intention and why you are interested in the organization name.
   - Include any relevant information, such as the programming language you are working with and how the organization name aligns with your project.

3. **Wait for Response:**
   - After submitting the form, GitHub Support will review your request and get back to you via email.
   - Be patient, as the response time may vary.

4. **Terms of Service:**
   - Ensure that your request aligns with GitHub's [Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service), and you are requesting the name for legitimate and non-infringing purposes.

**Reply by rvirding** [reply]
Thank you, I will try that!

**Comment by rvirding** [comment]
OK, I have tried that and now have a ticket. Let's hope I get an answer. As I mentioned the org Luerl is completely empty in every respect which makes it a bit difficult. :smile:

**Reply by Riu** [reply]
@rvirding did you receive an answer? What was the end of the case?

**Reply by sphildreth** [reply]
@rvirding did you receive an answer? What was the end of the case? I am facing the same situation.
